buildingmotif/api/app.py

```diff
@@ -69,11 +69,6 @@ def create_app(DB_URI):
 if __name__ == "__main__":
     """Run API."""
     db_uri = os.getenv("DB_URI")
-    if db_uri is None:
-        # If config doesn't exist, this is considered a third party import and module cant be found.
-        import configs as building_motif_configs  # type: ignore # isort:skip
-
-        db_uri = building_motif_configs.DB_URI
 
     app = create_app(db_uri)
```

Comment:
Do we want to throw an error here if the `db_uri` is not found? We could also have a default value of an in memory db or sometthing.

Reply:
I'd rather not default to anything.